explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -229,16 +288,13 @@ function AccountsCard({ signature }: Props) {
     }
   }, [signature, details, status, fetchDetails]);
 
-  if (!status?.data?.info) {
+  if (autoRefresh === AutoRefresh.BailedOut || !status?.data?.info) {
     return null;
-  } else if (!details) {
-    return (
-      <ErrorCard
-        retry={refreshStatus}
-        text="Details are not available until the transaction reaches MAX confirmations"
-      />
```

Comment:
Why did you decide to remove this state? I liked that it communicated to a user why they weren't seeing the full details for a transaction.

Reply:
We discussed this one earlier. We decided to show the loading card instead of having a "details are not available" w/ the retry button, since a auto refresh would be in process.

Reply:
Another option would be to show the message "Details are not available until the transaction reaches MAX confirmations" with a loading indicator.

Reply:
Went ahead and added that back!

Reply:
Ah ok I missed your reply here:

https://github.com/solana-labs/solana/pull/11747#discussion_r476098342
> So basically, there are three overall states: transaction not found, transaction hasn't reached max (auto refresh active), transaction has max confirmations.
> 
> The "details are not available until the transaction reaches MAX confirmations" error card would be eliminated.

I think there is one more state when we bail out from auto refresh. I think it would be sufficient to show this error card when the auto refresh is bailed out. 

> Another option would be to show the message "Details are not available until the transaction reaches MAX confirmations" w/o retry (with a loading indicator?).

But this sounds good too! Up to you. Dropping the retry in either case sounds good

Reply:
Oh I see you went with showing the message during auto refresh, nice!